WcaOnRails/app/javascript/edit-schedule/EditVenue/index.jsx

```diff
@@ -110,6 +117,24 @@ const NameInput = ({name, actionHandler}) => (
   </Row>
 );
 
+const CountryInput = ({value, onChange}) => (
+  <Row>
+    <Col xs={3}>
+      <span className="venue-form-label control-label">Country:</span>
+    </Col>
+    <Col xs={9}>
+      <select
+        className="form-control"
+        value={value}
+        onChange={onChange}
+      >
+        <option key="" value="" />
+        {countries.map(country => <option key={country.iso2} value={country.iso2}>{country.name}</option>)}
```

Comment:
Nit: this would benefit from multiple lines ^^

Reply:
Ahh, we started using `prettier` at work, so I have gotten lazy about formatting :(

Reply:
(fixed!)